x/wasm/types/types.go

```diff
@@ -423,3 +423,14 @@ func isSubset(super, sub []string) bool {
 	}
 	return matches == len(sub)
 }
+
+// AllAuthorizedAddresses returns the list of authorized addresses. Can be empty.
```

Comment:
This would be included in the #1189 backport. It is easier track changes separately for a clean history